api/v1alpha1/edgeworkload_webhook.go

```diff
@@ -101,6 +101,12 @@ func (r *EdgeWorkload) validate() error {
 		} else {
 			containersNames[container.Name] = struct{}{}
 		}
+
+		for _, port := range container.Ports {
+			if port.HostPort == 9100 {
+				notValidPaths = append(notValidPaths, containersMsg(container, "ports"))
```

Comment:
Which message will be presented in a case of failure?

Reply:
`the following paths in podSpec are not supported and should be removed: containers[container.Name].hostPort`

Reply:
So this error message isn't accurate. It should say something like: "HostPort 9100 is reserved for internal use on the device and cannot be set for user workloads"

Reply:
done